### DISCUSSION TITLE
What Do You Need

### DISCUSSION THREAD

**Discussion by jdhitsolutions** [jdhitsolutions/PSBluesky#1] [Ideas]
What sort of things would you like to be able to do with BlueSky from PowerShell?

**Comment by 4IoTMirko** [comment]
It would be great to have a way of interacting with messages, quite similar to what is available with posts right now. It could be useful to generate notifications or as a bucket to receive output from jobs running somewhere instead of sending mails, or just checking if your contacts responded to a query, etc.

**Reply by jdhitsolutions** [reply]
I haven't used direct messages much yet, but this is a reasonable request.
